### ISSUE TITLE
Course: An Intro to WP-CLI

### ISSUE THREAD

**Issue by courtneyr-dev** [WordPress/Learn#632]
https://make.wordpress.org/cli/handbook/

https://wp-cli.org/

https://wordpress.tv/?s=wp-cli

https://docs.google.com/document/d/1E8dmWj7QnXpRERUF1wY0NI4cb2Q4U_q6hHILK8OH5yY/edit#heading=h.xnh9qo4z17lc

CHECKLIST: Development Checklist
- [ ] Draft lesson plan
- [ ] Copy edit
- [ ] Style guide review
- [ ] Instructional Review
- [ ] Final review
- [ ] Publish
- [ ] Description and Objectives finalized
- [ ] Gather links to Support and Developer Docs
- [ ] Define several SEO keywords to use in the article and where they should be prominently used
- [ ] Consider any MarComms (marketing communications) resources and link to those
- [ ] Review any related material on Learn

**Comment by rosswintle** [comment]
@courtneyr-dev Did you want to assign this to me? I'm still interested in it. There is a draft lesson plan. I was kinda waiting for a decision on what format is best, but then your formats weren't properly defined.

Looks like you now have "workshops" for short videos and "courses" for groups of "lesson" content.

I hope to have some contribution time, so let me know what's next.

**Comment by courtneyr-dev** [comment]
I'll help chunk this down into individual lesson plans/workshops that can roll up into a full course soon.

**Comment by rosswintle** [comment]
@courtneyr-dev Cool - please do assign individual tasks to me if you need things planned out and executed. Thank you! 🙏

**Comment by courtneyr-dev** [comment]
https://github.com/WordPress/Learn/issues/751 next step :)

**Comment by rosswintle** [comment]
I'm confused about why a lesson plan is the next step.

Ideally, with my limited contribution time, I would:

- Finish outlines of the videos for a short course
- Have that reviewed
- Created and have reviewed detailed scripts (if this is required - I tend to work from prompts rather than fixed scripts)
- Record
- Pass over for editing (or use suggested tools and materials to do this myself)

Should I just continue building out the [course outline](https://docs.google.com/document/d/1E8dmWj7QnXpRERUF1wY0NI4cb2Q4U_q6hHILK8OH5yY/edit#heading=h.xnh9qo4z17lc)?

Also, is the checklist at the top of this issue correct? It looks, to me, like the checklist for a "lesson plan"?

**Comment by digitalchild** [comment]
@rosswintle I did a 30-minute talk on wp-cli at WordCamp Brisbane back in 2015. It's been re-uploaded to WordPress.tv today. Would this be helpful ? I can also help with a lesson plan/written version.

**Comment by rosswintle** [comment]
Right - I'm gonna totally skip the Training Team processes 'cos I'm so confused, and just say what I think we should do here.

1) We should make a short video tutorial as an intro to WP-CLI - does this need a separate GH issue?
2) We should make an outline for a more in-depth "course".
3) @jonathanbossenger has suggested doing a video workshop too, which I'm happy with and I guess it would probably be an interactive version of the video tutorial from 1) 

I think we can progress the intro video (1) and the course plan (initial work on 2) in parallel.

[My document](https://docs.google.com/document/d/1E8dmWj7QnXpRERUF1wY0NI4cb2Q4U_q6hHILK8OH5yY/edit#) is still a good starting point for all of this.

I shall do some work on that, but it may need to be split out into two docs: one for 1) and one for 2)

So: Action me to work on the doc, though I'm also happy for @digitalchild to contribute if they want to.

@jonathanbossenger Do you have a view on making a separate issue for a video tutorial?

**Comment by hlashbrooke** [comment]
Steps 1-3 above make sense to me as the way forward here! A video tutorial as an intro to WP-CLI would require a new issue in this repo to track it. Thanks for jumping into this @rosswintle!

**Comment by rosswintle** [comment]
Thanks, @hlashbrooke - I opened #829

**Comment by kaitohm** [comment]
Closing in favor of #829
